eeg_face13/derivatives/lossless/sub-s02/eeg/sub-s02_task-faceO_eeg_qcr_annotations.json

```diff
@@ -0,0 +1,23 @@
+{
+	"Description": "Lossless state channel and component annotations.",
+	"IntendedFor": "derivatives/lossless/sub-s02/eeg/sub-s02_task-faceO_eeg_qcr.edf",
```

Comment:
hmm ... I don't think the source suffix should be the part of the filename. Can you check?